core/app/modules/import-export/assets/js/hooks/use-link/use-link.js

```diff
@@ -0,0 +1,18 @@
+export default function useLink() {
+	return {
+		url: {
+			goPro: 'https://elementor.com/pro/',
+		},
+		action: {
+			backToDashboard: () => {
+				if ( window.top === window ) {
+					// Directly.
+					window.top.location = elementorAppConfig.return_url;
+				} else {
+					// Iframe.
+					window.top.$e.run( 'app/close' );
+				}
```

Comment:
Export this functionality to util and use it wherever needed

Reply:
This is the util (the hook itself).

Reply:
Done.